xclim/indices/_conversion.py

```diff
@@ -917,12 +918,26 @@ def snow_depth_from_amount(
         If snow water equivalent (`swe` [m]) is provided instead, will be converted to `snw` before calculating.
     snr : xr.DataArray, optional
         Snow density [kg/m^3].
+    const: Quantified, optional
```

Comment:
```suggestion
    const: Quantified
```

`Optional` here suggests that the signature can take a `None` value (but that would fail if  `snr` is `None`). I think it's better to just ask for `Quantified`